deposit/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -54,14 +54,14 @@
     </bean>
 
     <bean id="jesqueConfig" class="net.greghaines.jesque.Config">
-        <constructor-arg value="${redis.host:localhost}" />
-        <constructor-arg value="${redis.port:6379}" />
-        <constructor-arg value="2000" />
-        <constructor-arg>
+        <constructor-arg value="${redis.host:localhost}" type="java.lang.String" />
```

Comment:
seems like a good idea, but just curious what advantage the typing of the constructor args gives us?

Reply:
After updating the version of jesque there was a second constructor that was getting selected over the correct one since they had the same number of parameters I guess, even though the first one was a Set instead of a String, so I needed to give it a bit more direction.